interface/patient_file/summary/demographics.php

```diff
@@ -473,200 +475,217 @@ function setMyPatient() {
 </head>
 
 <body class="body_top patient-demographics">
-
-<a href='../reminder/active_reminder_popup.php' id='reminder_popup_link' style='visibility: false;' class='iframe' onclick='top.restoreSession()'></a>
-
-<?php
-$thisauth = acl_check('patients', 'demo');
-if ($thisauth) {
-    if ($result['squad'] && ! acl_check('squads', $result['squad'])) {
-        $thisauth = 0;
-    }
-}
-if (!$thisauth) {
-    echo "<p>(" . htmlspecialchars(xl('Demographics not authorized'),ENT_NOQUOTES) . ")</p>\n";
-    echo "</body>\n</html>\n";
-    exit();
-}
-if ($thisauth): ?>
-
-<table class="table_header">
-    <tr>
-        <td>
-            <span class='title'>
-                <?php echo htmlspecialchars(getPatientName($pid),ENT_NOQUOTES); ?>
-            </span>
-        </td>
-        <?php if (acl_check('admin', 'super') && $GLOBALS['allow_pat_delete']) : ?>
-        <td style='padding-left:1em;' class="delete">
-            <a class='css_button iframe'
-               href='../deleter.php?patient="<?php echo htmlspecialchars($pid,ENT_QUOTES);?>'
-               onclick='top.restoreSession()'>
-                <span><?php echo htmlspecialchars(xl('Delete'),ENT_NOQUOTES);?></span>
-            </a>
-        </td>
-        <?php endif; // Allow PT delete
-        if($GLOBALS['erx_enable']): ?>
-        <td style="padding-left:1em;" class="erx">
-            <a class="css_button" href="../../eRx.php?page=medentry" onclick="top.restoreSession()">
-                <span><?php echo htmlspecialchars(xl('NewCrop MedEntry'),ENT_NOQUOTES);?></span>
-            </a>
-        </td>
-        <td style="padding-left:1em;">
-            <a class="css_button iframe1"
-               href="../../soap_functions/soap_accountStatusDetails.php"
-               onclick="top.restoreSession()">
-                <span><?php echo htmlspecialchars(xl('NewCrop Account Status'),ENT_NOQUOTES);?></span>
-            </a>
-        </td>
-        <td id='accountstatus'></td>
-        <?php endif; // eRX Enabled
-        //Patient Portal
-        $portalUserSetting = true; //flag to see if patient has authorized access to portal
-        if( ($GLOBALS['portal_onsite_enable'] && $GLOBALS['portal_onsite_address']) ||
+<nav class="navbar navbar-default navbar-fixed-top">
+    <div class="container-fluid">
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#patient-context-bar" aria-expanded="false">
+                <span class="sr-only"><?php echo xl("Toggle navigation"); ?></span>
```

Comment:
use xlt()
(and also don't worry if the phrase has translations exists; I run a perl script pipeline every once in a while to bring in new constants)